e2e-tests/admin/tests/worker-tags.spec.js

```diff
@@ -3,11 +3,10 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
-import { test } from '../playwright.config.js';
+import { test, authenticatedState } from '../../global-setup.js';
 import { expect } from '@playwright/test';
 import { customAlphabet } from 'nanoid';
```

Comment:
this is a non blocker- I was just curious why we use `nanoid` instead of the `generateId` method from our mirage? looks like both return alphanumeric chars

Reply:
Yeah this was a convention that was already being used, in practice it doesn't matter too much which one we use, we also could have used `uuid` and maybe should consolidate.

But note that this folder does not have access to the ember addons or helpers anymore so we'd just use the standard packages for it.

Reply:
@DhariniJeeva @ZedLi 

None blocking comment: nice catch about the usage of different libraries for similar purposes. After reading both comments, I think its a good topic to discuss outside the PR context, since it was not introduced within the PR but already present.
Mentioning because recently I am working on a similar consolidation to `uuid` ([ICU-15818](https://hashicorp.atlassian.net/browse/ICU-15818)) and this felt familiar.
If you don't mind I will bring this topic to our next tech time: discuss uuid, nanoid, ember-uuid, generateId and standard crypto.randomUUID. All these libraries/packages are being used on Boundary UI while we could consolidate.

Reply:
Yeah I agree we should definitely consolidate our usages, we have it scattered in quite a few places.